internal/cmd/headless_repl/rust-toolchain.toml

```diff
@@ -0,0 +1,11 @@
+[toolchain]
+channel = "1.60"
```

Comment:
Note that, per @Westonside, there's some dependency that depends on features in the later toolchain. We can probably pin to an earlier version of whatever it is, to avoid needing the newer compiler.

Reply:
We got the go-ahead from the edge team to bump libflux up to 1.63. Sounds like they resolved the cross-build problems on their end.

We can loop back and remove this toolchain file when we bump.